File: tfm-sgpfe-simulator-rneves/service_simulator/lib/modules/common/domain/entities/ticket.dart
import 'package:equatable/equatable.dart';
import 'package:meta/meta.dart';

class Ticket extends Equatable {
  final int id;
  final int number;
  final String letter;
  final String name;
  final int queueId;

  Ticket(
      {@required this.id,
      @required this.number,
      @required this.letter,
      @required this.name,
      @required this.queueId});

  @override
  List<Object> get props => [id, number, letter, name, queueId];
}

File: tfm-sgpfe-rneves/server/src/main/kotlin/isel/meic/tfm/fei/entities/Ticket.kt
package isel.meic.tfm.fei.entities

/**
 * Instituto Superior de Engenharia de Lisboa
 * Project: Smart Queue
 * @author Ronilton Neves 39643
 * @mentor: Paulo Pereira
 *
 * Represents the entry of the Db Ticket table
 */
data class Ticket(val id: Int, var number: Int, val letter : String, val serviceId: Int, val name: String, val postOfficeId : Int)
